integration-tests/run-allure.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+# Start Allure services & generate reports
+docker compose -f ${SCRIPT_DIR}/docker-compose.allure.yaml up -d
+
+# Wait for UI to become available
+while ! curl --fail --silent --head http://localhost:5252; do
+  sleep 1
+done
+
+sleep 10
+
+# Open browser at latest report
+open http://localhost:5252/allure-docker-service-ui/projects/default
```

Comment:
Can we add a command to rename the generated report? Because I need to upload the report everyday onto Confluence, and it complains if files have the same name. I had this in a local branch, which is now behind yours `mv allure-reports/email*.html allure-reports/TxMA-Team2-Test_Report-$(date +%d-%m-%Y).html`. Thanks